Mathlib/SetTheory/Game/Basic.lean

```diff
@@ -492,14 +492,14 @@ lemma neg_mul (x y : PGame) : -x * y ≡ -(x * y) :=
 
 @[simp]
 theorem quot_neg_mul (x y : PGame) : (⟦-x * y⟧ : Game) = -⟦x * y⟧ :=
-  game_eq (negMulRelabelling x y).equiv
+  game_eq (x.neg_mul y).equiv
 
 /-- `x * -y` and `-(x * y)` have the same moves. -/
 def mulNegRelabelling (x y : PGame) : x * -y ≡r -(x * y) :=
   (mulCommRelabelling x _).trans <| (negMulRelabelling _ x).trans (mulCommRelabelling y x).negCongr
 
 theorem quot_mul_neg (x y : PGame) : ⟦x * -y⟧ = (-⟦x * y⟧ : Game) :=
-  game_eq (mulNegRelabelling x y).equiv
+  game_eq (x.mul_neg y ▸ Setoid.refl _) -- Porting note: was `of_eq (x.mul_neg y)`
```

Comment:
`Setoid.refl` seems like somewhat of a code smell, is there some other lemma you can use instead?

Also, a porting note doesn't make sense since the porting from Lean 3 already finished!

Reply:
`game_eq (by rw [mul_neg])` or `game_eq (x.mul_neg y ▸ (equiv_iff_game_eq.mpr rfl))` also work here.

Reply:
(I'm not particurally sure why Setoid.refl is a code smell here, so I'll wait for a response before I substitute one of these in.)

Reply:
Does `game_eq (by rfl)` work? If not, then `game_eq (by rw [mul_neg])` seems like the cleanest approach.

The issue with using the `Setoid` API directly is that it writes things in terms of `Setoid.r`, whereas in our case we'd rather be working only with `AntisymmRel`. Any important theorems on setoids (like `game_eq`!) should generally be re-stated in terms of the actual structure we're using.

Reply:
By the way, our current reliance on setoid notation is the reason dot notation is broken on `Equiv`. That should hopefully get fixed when I redefine `Game` as the `Antisymmetrization` of `PGame`.